cmd/create.go

```diff
@@ -142,9 +137,16 @@ A full list of component types that can be deployed is available using: 'odo com
 			fmt.Printf("Please wait, creating %s component ...\n", componentName)
 			err = component.Build(client, componentName, applicationName, false, true, stdout)
 			checkError(err, "")
-			fmt.Printf("Component '%s' was created.\n", componentName)
-			fmt.Printf("To push source code to the component run 'odo push'\n")
 		}
+		ports, err := component.GetComponentPorts(client, componentName, applicationName, false)
```

Comment:
it doesn't really make sense to call this function with `additional` flag set to true.
You can just completely remove this argument from the function and always call `componentlabels.GetLabels` with `false`. Sorry this was what I meant with my previous comment 😇

Reply:
oh ok np :)